DEVELOPMENT.md

```diff
@@ -0,0 +1,318 @@
+<!-- spell-checker:ignore reimplementing toybox RUNTEST CARGOFLAGS nextest -->
+
+# Setting up local development environment
+
+For contributing rules and best practices please refer to [CONTRIBUTING.md](CONTRIBUTING.md)
+
+## Before you start
+
+For this guide we assume that you already have GitHub account and have `git` and your favorite code editor or IDE installed and configured to work with GitHub.
+Before you start working on coreutils, please follow these steps:
+
+1. Fork [coreutils repository](https://github.com/uutils/coreutils) to your GitHub account [ref:](https://docs.github.com/en/get-started/quickstart/fork-a-repo)
```

Comment:
```suggestion
1. Fork the [coreutils repository](https://github.com/uutils/coreutils) to your GitHub account [ref:](https://docs.github.com/en/get-started/quickstart/fork-a-repo)
```

I'd also like to have some more text as the link than "ref". Maybe `See [this GitHub guide](...) for more information.`